test/unit/org/apache/cassandra/db/CellTest.java

```diff
@@ -172,9 +172,6 @@ public void testValidate()
 
         // Invalid ttl
         assertInvalid(BufferCell.expiring(c, 0, -4, 4, bbs(4)));
-        // Cells with overflowed localExpirationTime are valid after CASSANDRA-14092
```

Comment:
Instead of just removing the tests, we could verify that they are rejected with `value (-1) is outside the range for an unsigned integer value`.

Reply:
Will do